packages/notebook/src/browser/contributions/notebook-actions-contribution.ts

```diff
@@ -180,6 +204,43 @@ export class NotebookActionsContribution implements CommandContribution, MenuCon
             execute: () => (this.shell.activeWidget as NotebookEditorWidget).redo()
         });
 
+        commands.registerCommand(NotebookCommands.CUT_SELECTED_CELL, this.editableCommandHandler(
+            () => {
+                const model = this.notebookEditorWidgetService.focusedEditor?.model;
+                const selectedCell = model?.selectedCell;
+                if (selectedCell) {
+                    model.applyEdits([{ editType: CellEditType.Replace, index: model.cells.indexOf(selectedCell), count: 1, cells: [] }], true);
+                    this.notebookClipboardService.copyCell(selectedCell);
+                }
+            }));
+
+        commands.registerCommand(NotebookCommands.COPY_SELECTED_CELL, {
+            execute: () => {
+                const model = this.notebookEditorWidgetService.focusedEditor?.model;
+                const selectedCell = model?.selectedCell;
+                if (selectedCell) {
+                    this.notebookClipboardService.copyCell(selectedCell);
+                }
+            }
+        });
+
+        commands.registerCommand(NotebookCommands.PASTE_CELL, {
+            isEnabled: () => !Boolean(this.notebookEditorWidgetService.focusedEditor?.model?.readOnly),
+            isVisible: () => !Boolean(this.notebookEditorWidgetService.focusedEditor?.model?.readOnly),
+            execute: (position: 'above' | 'below') => {
```

Comment:
Minor: `"below"` is actually never used as an argument. It's either `"above" | undefined`, which should be reflected in the type.
```suggestion
            execute: (position?: 'above') => {
```